### DISCUSSION TITLE
How to use websocket for live asr speaker diarzation.

### DISCUSSION THREAD

**Discussion by jarvisx17** [juanmc2005/diart#150] [Q&A] [answered]
can anybody help me on how to use web socket in diart because I'm not getting any response using websocket and pyaudio?

**Comment by juanmc2005** [comment, marked as answer]
Hi @jarvisx17, do the `diart.client` and `diart.server` CLI tools work with your installation?
Also, make sure you're using v0.7 where a lot of websocket issues were fixed.
